src/main/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -36,10 +38,16 @@ public data class YamlConfiguration constructor(
     internal val extensionDefinitionPrefix: String? = null,
     internal val polymorphismStyle: PolymorphismStyle = PolymorphismStyle.Tag,
     internal val encodingIndentationSize: Int = 2,
-    internal val breakScalarsAt: Int = 80
+    internal val breakScalarsAt: Int = 80,
+    internal val sequenceStyle: SequenceStyle = SequenceStyle.Block
```

Comment:
Could you please add a short description of this parameter in the documentation comment above?

Reply:
Something like this would be enough:

```
[sequenceStyle]: how to format sequences (lists or arrays) when encoding them as YAML
```